### ISSUE TITLE
Ready for prime time? (FreeCAD Addon Manager)

### ISSUE THREAD

**Issue by luzpaz** [iplayfast/CycloidGearBox#4]
Are you ready to add this to FreeCAD Addon Manager?

**Comment by iplayfast** [comment]
Not quite. There's an existing bug that I want to fix first. (I'm stuck). https://forum.freecadweb.org/viewtopic.php?f=22&t=67086&sid=81cf1006c2d7d907a5d17bad788b0f50 describes the issue. Basically I can create the initial files, but can't update them.

**Comment by iplayfast** [comment]
I think it's ready for primetime.
